runtime/calamari/tests/integrations_mock/integration_tests.rs

```diff
@@ -1113,27 +1116,10 @@ fn concrete_fungible_ledger_transfers_work() {
                 0
             );
 
-            // Transferring invalid asset ID should not work.
```

Comment:
`FungibleLedgerError::InvalidAssetId` still exists, why remove the test?

Reply:
There's no DummyAssetId anymore